.github/workflows/test.yaml

```
@@ -16,5 +16,58 @@ jobs:
+          name: dist
+          path: dist
+
+      - uses: ./.github/actions/setup-app
+
+      - name: Run ${{ matrix.tests.type }}
+        run: |
+          npm run test:ember -- ${{matrix.tests.commandArgs}} --path=dist --wmmf
+
+      - name: Job Summary
+        if: always()
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)